src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1434,6 +1442,8 @@ def create_oracledb_connection():
         )
         struct_schema = partitioner.schema
         partitioned_queries = partitioner.partitions
+        telemetry_json_string["DBMS_type"] = partitioner.dbms_type
```

Comment:
nit: I think you can lower case the "DBMS" to make the key casing consistent
```suggestion
        telemetry_json_string["dbms_type"] = partitioner.dbms_type
```